.github/workflows/push.yml

```diff
@@ -23,23 +23,65 @@ jobs:
         cd docs
         make html-author-mode
         grep -r "page unresolved" target/staging/docs/ && exit 1 || echo 'ok'
-  build-and-test-scala:
+
+  build-and-test-core:
+    name: "Cloudflow Core"
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        java: [ '8', '11', '13' ]
     steps:
-    - uses: actions/checkout@v2
 
-    - name: Setup environment
-      run: |
-        make -C .github/workflows/setup setup-local-all
```

Comment:
good catch we can remove the entire `Makefile`:
https://github.com/lightbend/cloudflow/blob/main/.github/workflows/setup/Makefile

Reply:
oh, it's done, nice, thanks!